src/javascript/binary/base/binary_pjax.js

```diff
@@ -133,7 +133,7 @@ const BinaryPjax = (() => {
     const handlePopstate = (e) => {
         const url = e.state ? e.state.url : window.location.href;
         if (url) {
-            processUrl(url, true);
+            handleClick(url, true);
```

Comment:
Does it break the browser's back button? (`handleClick()` expects the first parameter to be the event, otherwise does nothing)

Reply:
@binary-hamedanchi It actually breaks the back button functionality, let me see if I can make it work.